### PR TITLE
Req edit

### PR DESCRIPTION
**Pull request by aniketrege**
Updated requirements.txt file for pip installation

### CODE REVIEW

File: HowLong/HowLong.py [hunk 1]
```diff
@@ -89,12 +97,12 @@ def run(self):
         process = Process(pid=self.pid,command=self.command)
         readable_command = process.command
         start_time = process.start_time
-        logging.debug("Running " + readable_command)
+        logging.debug(colored("Running " + readable_command), 'green')
```
Comment (mattjegan) [inline HowLong/HowLong.py]: This needs to be `logging.debug(colored("Running " + readable_command, 'green'))`

File: HowLong/HowLong.py [hunk 2]
```diff
@@ -89,12 +97,12 @@ def run(self):
...
         while process.is_running():
             sleep(self.timer_interval)
             elapsed_time = (time() - start_time) * 1000
-            logging.info(red(str(timedelta(milliseconds=elapsed_time))))
-        logging.debug("Finished " + readable_command)
+            logging.info(colored(str(timedelta(milliseconds=elapsed_time)), 'blue'))
+        logging.debug(colored("Finished " + readable_command), 'red')
```
Comment (mattjegan) [inline HowLong/HowLong.py]: This needs to be: `logging.debug(colored("Finished " + readable_command, 'red'))`

**Review by mattjegan — CHANGES_REQUESTED** [review]
Looks pretty good to me, just a few typos that need fixing before I can merge. Thanks 😄

**Comment by aniketrege** [comment]
Hey Matt, I've fixed those typos for you

**Review by mattjegan — APPROVED** [review]

**Pull request merged**